src/jltypes.c

```diff
@@ -2069,43 +2069,45 @@ void jl_init_types(void) JL_GC_DISABLED
-                            "slotflags",
-                            "slottypes",
-                            "rettype",
-                            "parent",
-                            "min_world",
-                            "max_world",
-                            "inferred",
-                            "inlineable",
-                            "propagate_inbounds",
-                            "pure"),
-                        jl_svec(17,
-                            jl_array_any_type,
-                            jl_any_type,
-                            jl_any_type,
-                            jl_array_uint8_type,
-                            jl_any_type,
-                            jl_any_type,
-                            jl_array_symbol_type,
-                            jl_array_uint8_type,
-                            jl_any_type,
-                            jl_any_type,
-                            jl_any_type,
-                            jl_long_type,
-                            jl_long_type,
-                            jl_bool_type,
-                            jl_bool_type,
-                            jl_bool_type,
-                            jl_bool_type),
-                        0, 1, 17);
+                        jl_perm_symsvec(18,
+                                        "code",
+                                        "codelocs",
+                                        "ssavaluetypes",
+                                        "ssaflags",
+                                        "method_for_inference_limit_heuristics",
+                                        "linetable",
+                                        "slotnames",
+                                        "slotflags",
+                                        "slottypes",
+                                        "rettype",
+                                        "parent",
+                                        "edges",
+                                        "min_world",
+                                        "max_world",
+                                        "inferred",
+                                        "inlineable",
+                                        "propagate_inbounds",
+                                        "pure"),
+                        jl_svec(18,
+                                jl_array_any_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_array_uint8_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_array_symbol_type,
+                                jl_array_uint8_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_any_type,
+                                jl_ulong_type,
+                                jl_ulong_type,
```

Comment:
:+1: I don't think we used to have the ulong type available here previously. Can we change all of the other similar slots too now?